### PR TITLE
Enable Tuple with duplicate keys

### LINKED ISSUE #213: Implement PartiQL `Tuple`
This issue is to track the effort for enabling PartiQL tuples to store values with duplicate keys, e.g. `{'a': 1, 'a': 2', 'b': 1, 'a': 1}`.

### DoD
- Available options are evaluated as discussed with the team.
- The selected option is PR'ed and merged.

### PR DESCRIPTION
**Pull request by am357**
*Issue #, if available:*

Closes #213 

*Description of changes:*

This PR includes the required changes to allow duplicate keys in `Tuple` to conform
with PartiQL definition of `Tuple` type as oppose to `Map` with unique keys.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-value/src/lib.rs [hunk 1]
```diff
@@ -566,7 +567,6 @@ impl Ord for Value {
             (Value::List(_), _) => Ordering::Less,
             (_, Value::List(_)) => Ordering::Greater,
 
-            (Value::Tuple(l), Value::Tuple(r)) => l.cmp(r),
```
Comment (jpschorr) [inline partiql-value/src/lib.rs:569]: why is this removed?

Comment (jpschorr) [inline partiql-value/src/lib.rs:569]: Why is this removed?

File: partiql-value/src/lib.rs [hunk 2]
```diff
@@ -886,90 +886,97 @@ impl Hash for Bag {
...
+            .fold(Tuple::new(), |mut acc: Tuple, (attr, val)| {
+                acc.insert(attr, val.into());
+                acc
+            });
+
+        out
     }
 }
 
-impl Ord for Tuple {
```
Comment (jpschorr) [inline partiql-value/src/lib.rs:935]: Why is `Ord` removed?

Comment (am357) [reply partiql-value/src/lib.rs:935]: Was a miss, fixed in the new commit.

Comment (am357) [reply partiql-value/src/lib.rs:935]: Re-visiting this to ensure it conforms with the spec. if so, will re-request a review, thanks.

**Review by jpschorr — APPROVED** [review]
